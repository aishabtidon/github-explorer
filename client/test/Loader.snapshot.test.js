const { expect, test } = require('@jest/globals');

// Simple snapshot test for Loader component functionality
test('Loader component snapshot', () => {
  // Test the Loader component's basic structure
  const loaderData = {
    text: 'Loading…',
    component: 'Loader',
    props: {
      role: 'img',
      'aria-label': 'spinner'
    }
  };
  
  expect(loaderData).toMatchSnapshot();
});

test('Loader with custom text snapshot', () => {
  const loaderData = {
    text: 'Loading user…',
    component: 'Loader',
    props: {
      role: 'img',
      'aria-label': 'spinner'
    }
  };
  
  expect(loaderData).toMatchSnapshot();
});
