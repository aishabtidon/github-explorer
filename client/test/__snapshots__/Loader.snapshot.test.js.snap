// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component snapshot 1`] = `
{
  "component": "Loader",
  "props": {
    "aria-label": "spinner",
    "role": "img",
  },
  "text": "Loading…",
}
`;

exports[`Loader with custom text snapshot 1`] = `
{
  "component": "Loader",
  "props": {
    "aria-label": "spinner",
    "role": "img",
  },
  "text": "Loading user…",
}
`;
